### DISCUSSION TITLE
Enable code editor by default

### DISCUSSION THREAD

**Discussion by yuvarajperumal** [facebook/lexical#5004] [Q&A] [answered]
I am looking for a simple editor which just accepts the code. I dont want any of the other rich editor stuffs.
I dont know how to configure it directly. In all the examples it is been configured through a toolbar. 
Is there a way I can create a simple code editor with lexical without any other complex configurations?

**Comment by Benbinbin** [comment, marked as answer]
I don't think Lexical is a powerful editor for Code, maybe using the specific framework for building code editor for web like [CodeMirror](https://codemirror.net/)

Or use the framework to build more powerful code editor which can run JS script online, like [sandpack](https://sandpack.codesandbox.io/) or [webcontainers](https://webcontainers.io/)
